### DISCUSSION TITLE
Please help to update the Discord link invitation?

### DISCUSSION THREAD

**Discussion by sidoqi** [DioxusLabs/dioxus#2414] [General]
I want to join the server, but it seems the link was expired. Please help to update. Thanks!

**Comment by ealmloff** [comment]
Are you using [this invite](https://discord.gg/XgGxMSkvUM)? It seems to be working. People have joined with that link earlier today

**Reply by sidoqi** [reply]
I'm using the one in the site I think and it's already expired.

Thanks btw!

**Reply by ealmloff** [reply]
The invite on the homepage works
